utils/frame/try-runtime/cli/src/lib.rs

```diff
@@ -274,6 +279,8 @@ where
 		max_runtime_instances,
 	);
 
+	check_spec_name::<Block>(shared.url.clone(), config.chain_spec.name().to_string()).await;
```

Comment:
We are already checking L204. Do we need to check again?

Reply:
yes, they are different functions and different call paths.